eth_keys/utils/asn1.py

```diff
@@ -0,0 +1,112 @@
+# Non-recoverable signatures are encoded using an asn1 sequence of two integers
```

Comment:
Nitpick regarding the module name: ASN1 seems to be just the language to describe the schema, the encoding is called DER.

Reply:
Ah, thanks for pointing that out. Will rename.